modules/nw-about-automatic-egress-ips.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * networking/enabling-automatic-egress-ips.adoc
+
+[id="about-automatic-egress-ips-{context}"]
+= About automatic egress IPs
+
+You can assign egress IP addresses to namespaces by setting the `egressIPs`
+field to the `NetNamespace` resource. With fully automatic egress IPs, you can
+set the `egressCIDRs` field of each node's `HostSubnet` resource to indicate the
+range of egress IP addresses that can be hosted. Namespaces that have requested
+egress IP addresses are matched with nodes that are able to host those egress IP
+addresses, then the egress IP addresses are assigned to those nodes.
+
+High availability is automatic. If a node hosting egress IP addresses
+goes down and there are nodes that are able to host those egress IP addresses,
```

Comment:
I'm wondering if we can use alternate wording for "goes down" here, as I'm not sure it will translate well.